app/Schema/Mysql.php

```diff
@@ -1104,8 +1104,8 @@ function version_46(PDO $pdo)
     $rq->execute(array('is duplicated by', get_last_insert_id($pdo)));
     $arq->execute(array(get_last_insert_id($pdo), 'duplicates'));
 
-    $rq->execute(array('is a parent of', 0));
-    $rq->execute(array('is a child of', get_last_insert_id($pdo)));
+    $rq->execute(array('is a child of', 0));
+    $rq->execute(array('is a parent of', get_last_insert_id($pdo)));
```

Comment:
This change will be effective only for people who are creating the database for the first time. It would be better to run a migration to swap the IDs.

Or better, do what I mentioned here: https://github.com/kanboard/kanboard/pull/4888#discussion_r715898833